### DISCUSSION TITLE
Every good project needs a domain

### DISCUSSION THREAD

**Discussion by emb4fun** [stnolting/neorv32#208] [Show and tell]
And before the domain grabbers got a chance, I was faster:

https://neorv32.de
https://neorv32.com
https://neorv32.org

Best regards,
Michael

**Comment by stnolting** [comment]
This is really great (and looks very nice)!
Thank you so much!!! :)

> https://neorv32.org

About time to turn this repository into a GitHub organization, right? 😅

**Reply by umarcor** [reply]
Note that it should be possible to configure those domains as a redirect to stnolting.github.io/neorv32. Hence, the content might be updated from this repo through GitHub Pages, but it would be accesible with shorter URLs.

**Comment by emb4fun** [comment]
Yes, this is technically possible, but here I have concerns about liability since the
domain and the GitHub content could be seen as a unit. And should there be any problems
with the content, it would first go to the domain owner.

The domains were currently paid for one year, a domain transfer to another person after
the expiry of the term or earlier  will I agree if that should be desired.

In the current implementation, it should be clear that these domains are separated from
the GitHub content. That's why I put the imprint here too.
